src/components/DiffView/index.spec.tsx

```diff
@@ -168,35 +156,34 @@ describe(__filename, () => {
   });
 
   it('renders hunks with separators', () => {
-    const diffs = parseDiff(diffWithDeletions);
-    const root = renderWithLinterProvider({ diffs });
+    const diff = parseDiff(diffWithDeletions)[0];
+    const root = renderWithLinterProvider({ diff });
 
     // Simulate the interface of <Diff />
     const children = root.find(Diff).prop('children');
-    const diff = shallow(<div>{children(diffs[0].hunks)}</div>);
+    const diffComponent = shallow(<div>{children(diff.hunks)}</div>);
```

Comment:
`diffInstance`? or `wrapper?` or `diffWrapper`?

`Component` makes me think that it's something to instantiate.